### DISCUSSION TITLE
An ability to display local max value for Y axis

### DISCUSSION THREAD

**Discussion by y9san9** [patrykandpatrick/vico#403] [Ideas]
https://github.com/patrykandpatrick/vico/assets/46930374/313e27af-9293-4665-8f1d-61bf92a55a5f

Is there a possibility to adjust max value on the Y axis? If at some point I had 12:00:00 it will be inconvenient to see such max value for axis. 

The daily value is often between 01:00:00 and 01:00:00 and it would be fantastic if your library support displaying on Y axis the biggest visible value.

P.s. your library is great, thank you

**Comment by y9san9** [comment]
Actually #269 could solve my issue, but not in the best way (the expected behaviour is described in the post and a lot of libraries for web/python can do that)

**Comment by patrickmichalik** [comment]
Hello! Thanks for the kind words and the suggestion. Due to the complexity of this change, it’s not planned for the near future, but we plan on adding such functionality down the line.
